tests/README.rst

```diff
@@ -0,0 +1,68 @@
+.. FogLAMP test scripts describes how to FogLAMP scripted tests are organised and how to write the scripted tests
+
+.. |br| raw:: html
+
+   <br />
+
+.. Links
+
+.. Links in new tabs
+
+.. |pytest docs| raw:: html
+
+   <a href="https://docs.pytest.org/en/latest/contents.html" target="_blank">pytest</a>
+
+.. _Unit: unit\\README.rst
+.. _Integration: integration\\README.rst
+.. _System: system\\README.rst
+
+.. =============================================
+
+********************
+FogLAMP Test Scripts
+********************
+
+FogLAMP scripted tests are classified into three categories:
+
+- `Unit`_ - Tests that checks the expected output of a code block.
+- `Integration`_ - Tests that checks the integration of different FogLAMP units that work as a single component.
+- `System`_ - Tests that checks the end to end flows in FogLAMP
+
+
+Running FogLAMP scripted tests
+==============================
+
+Test Prerequisites
+------------------
+
```

Comment:
I believe there are more prerequisites than this - for example the tests seem to need a system running before the tests are run. Do we not need to set PYTHONPATH before running the tests?

We may consider having a script to bundle up all the things we need to do to run test in the same way as we do with scripts/foglamp start